{"uuid":"9d2ac6bc-e736-4219-9b32-e0821629431a","historyId":"1260e2e8e70ea4e78bb1e04188dca3f3","fullName":"Telemedicine_MS.Telemedicine_Enrollee_Template.Get_All_Consultation","labels":[{"name":"package","value":"Telemedicine_MS.Telemedicine_Enrollee_Template"},{"name":"testClass","value":"Telemedicine_MS.Telemedicine_Enrollee_Template"},{"name":"testMethod","value":"Get_All_Consultation"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Rest_Test"},{"name":"subSuite","value":"Telemedicine_MS.Telemedicine_Enrollee_Template"},{"name":"host","value":"RELIANCEHMOs-MacBook-Pro.local"},{"name":"thread","value":"5125@RELIANCEHMOs-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Get All consultations"},{"name":"severity","value":"critical"}],"links":[],"name":"Get_All_Consultation","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Successf] but found [Success]","trace":"java.lang.AssertionError: expected [Successf] but found [Success]\n\tat org.testng.Assert.fail(Assert.java:110)\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\n\tat org.testng.Assert.assertEquals(Assert.java:131)\n\tat org.testng.Assert.assertEquals(Assert.java:655)\n\tat org.testng.Assert.assertEquals(Assert.java:665)\n\tat Telemedicine_MS.Telemedicine_Enrollee_Template.Get_All_Consultation(Telemedicine_Enrollee_Template.java:38)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:808)\n\tat org.testng.TestRunner.run(TestRunner.java:603)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:429)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:326)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1169)\n\tat org.testng.TestNG.runSuites(TestNG.java:1092)\n\tat org.testng.TestNG.run(TestNG.java:1060)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","description":"Ensure we can get all Consultations","steps":[],"attachments":[],"parameters":[],"start":1660222549630,"stop":1660222553253}